### ISSUE TITLE
PDF printing on Brother QL-710w ignores height on OSX

### ISSUE THREAD

**Issue by lichtamberg** [qzind/tray#299]
Even when I specify the height, the label which gets printed out is always the default 29mm high (when using a 62mm wide paper roll).

This worked in 2.0.5?

**Comment by tresf** [comment]
This sounds strikingly familiar to https://github.com/qzind/tray/issues/174 and it's quite concerning if this is the case, we'd have to roll back this patch and provide an immediate 2.0.6-1 release for MacOS.

Can you paste your Java version?  This should have been fully tested but we did not necessarily regression-test agaisnt older Java versions.

**Comment by lichtamberg** [comment]
Sure: 

```shell
java -version
java version "1.8.0_131"
Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)
```

Tried on OSX 10.12.6 Sierra with latest Firefox + Chrome.
Please let me know if you need some more information / logs / versions..

**Comment by tresf** [comment]
@lichtamberg thanks, this is helpful.  At the type of writing this, we can't update to Java 1.8.0_u162 because the patch is SDK-only, the runtime is only at u161 currently.

We'll downgrade Java and see if we can reproduce.

**Comment by klabarge** [comment]
Using PDFwriter on macOS, there is a difference in the page size between 2.0.5 and 2.0.6. However, the actual printed content seems to be the same

This was reproducible on Java 1.8u162 ([public download available here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html)) as well as Java 1.8u131

I used `size: {width: 4, height: 6}` for the below prints

* QZ Tray 2.0.6 did not change the page size

   ![image](https://user-images.githubusercontent.com/12505463/38392419-6e047fa0-38f5-11e8-8d83-989e6a90b92b.png)

* QZ Tray 2.0.5 changed the page size

  ![image](https://user-images.githubusercontent.com/12505463/38392531-d2a2a888-38f5-11e8-9d6b-f504f7942c12.png)

* QZ Tray 2.0.6 without page size specified

   ![image](https://user-images.githubusercontent.com/12505463/38392776-b2e9ad60-38f6-11e8-87ed-a79905f616a9.png)

**Comment by klabarge** [comment]
I reverted https://github.com/qzind/tray/commit/28736666aa73f707e4f919e63f63b6ad74834ee7 and printing was identical to 2.0.5

**Comment by tresf** [comment]
Turns out to be a version parsing bug.  `Constants.JAVA_VERSION` is returning `1.8.0` instead of `1.8.0-162`, causing this problem.  I'm on the case.

**Comment by tresf** [comment]
Ok, [`jsemver`](https://github.com/zafarkhaja/jsemver) requires the format to be in `1.8.0+162` (notice the `+` instead of the `-`).  Patched via  0d91e06.  Installer available here: https://qz.io/download

@lichtamberg thanks for the bug report, greatly appreciated.
